doc/nrf/releases_and_maturity/migration/migration_guide_3.1.rst

```diff
@@ -40,6 +40,40 @@ This section describes the changes related to libraries.
 
 .. _migration_3.1_recommended:
 
+Build system
+============
+
+.. toggle::
+
+   * In sysbuild, the following CMake extensions have been removed:
+
+     * ``sysbuild_dt_nodelabel``
+     * ``sysbuild_dt_alias``
+     * ``sysbuild_dt_node_exists``
+     * ``sysbuild_dt_node_has_status``
+     * ``sysbuild_dt_prop``
+     * ``sysbuild_dt_comp_path``
+     * ``sysbuild_dt_num_regs``
+     * ``sysbuild_dt_reg_addr``
+     * ``sysbuild_dt_reg_size``
+     * ``sysbuild_dt_has_chosen``
+     * ``sysbuild_dt_chosen``
+
+     From now on, you can use the regular devicetree extensions instead, such as ``dt_nodelabel``.
+     To specify the sysbuild image, the ``IMAGE`` argument is substituted with ``TARGET``.
```

Comment:
```suggestion
     You must now use standard devicetree extensions, such as ``dt_nodelabel``.
     To specify the sysbuild image, use the ``TARGET`` argument instead of ``IMAGE``.
```